LICENSE.md

```diff
@@ -47,11 +47,10 @@ http://hg.grauw.nl/grauw-lib
 
 https://github.com/cujojs/when
 
-> Open Source Initiative OSI - The MIT License
+>  Licensed under the MIT License at:
+>  http://www.opensource.org/licenses/mit-license.php
 >
-> http://www.opensource.org/licenses/mit-license.php
->
-> Copyright (c) 2011 Brian Cavalier
+>  MIT License (c) copyright B Cavalier &amp; J Hann
```

Comment:
oh oops, I was going by what was in Cesium: 
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/ThirdParty/when.js